styles.css

```diff
@@ -0,0 +1,135 @@
+body {
+  margin: 0;
+}
+
+.menu-btn {
+  width: 20px;
+  height: 20px;
+  background-color: transparent;
+  border: none;
+}
+
+#menu-container {
+  height: 100vh;
+  width: 100%;
```

Comment:
[OPTIONAl] I can see that `width: 100%` is a very common rule you declared all through your CSS. In order to not repeat yourself, consider making a separate class for this rule

Like this.

```
.full-width { 
    width: 100%;
}
```

You can then apply this in your HTML tags